flash/core/utilities/imports.py

```diff
@@ -85,9 +86,7 @@ def _compare_version(package: str, op, version) -> bool:
 
 if Version:
     _TORCHVISION_GREATER_EQUAL_0_9 = _compare_version("torchvision", operator.ge, "0.9.0")
-    _PYSTICHE_GREATER_EQUAL_0_7_2 = _compare_version("pystiche", operator.ge, "0.7.2")
 
-_IMAGE_STLYE_TRANSFER = _PYSTICHE_AVAILABLE
 _TEXT_AVAILABLE = _TRANSFORMERS_AVAILABLE
 _TABULAR_AVAILABLE = _TABNET_AVAILABLE and _PANDAS_AVAILABLE
```

Comment:
this makes it confusing, how about if you have installed only one... then anyone using the wrapper shall carefully set that you really need both...

Reply:
If just one of the tabular dependencies is installed and you try to use the TabularClassifier then it will error saying it needs all tabular dependencies to run. I don't think there are parts of the tabular classifier that you can use without installing those dependencies.

Reply:
so you say that ll tabular always need both dependencies, right?

Reply:
Yeah, if you want to train a model then it will need both tabnet and pandas